openshift-with-appstudio-test/e2e/util.go

```diff
@@ -188,31 +189,41 @@ func setup(t *testing.T, ta *testArgs) *testArgs {
 	if err != nil {
 		debugAndFailTest(ta, err.Error())
 	}
+	owner := os.Getenv("QUAY_USER")
```

Comment:
should we reuse the `QUAY_USERNAME` already used in the dev flow?

Reply:
Actually looking at the e2e repo I think this should probably be 'QUAY_E2E_ORGANIZATION'